src/content/docs/security/basicsecurity.mdx

```diff
@@ -0,0 +1,47 @@
+---
+description: Managing Your System(s)
+title: Basic Security
+slug: sec/basicsecurity
+---
+
+### Malicious Actors
+
+As with any public server, your IP is exposed in some way to the public.
+
+Malicious Actors (Hackers) are constantly scanning networks around the world in the hopes to find a vulnerable system they can attack.
+
+Vulnerabilities come in many different shapes and sizes and chances are you may have of heard or seen them in the past.
+
+**Some Examples**
+<ul class="pl-5 list-disc">
+    <li>CVE-2021-44228 - Log4Shell (Log4J)</li>
+    <li>CVE-2021-38003 - Google Chrome Vulnerability</li>
+    <li>CVE-2022-21658 - RCE in Minecraft: Java Edition</li>
+    <li>CVE-2023-29347 - Steam Client Vulnerability</li>
```

Comment:
[CVE-2022-21658](https://nvd.nist.gov/vuln/detail/CVE-2022-21658) isn't a Minecraft RCE and [CVE-2023-29347](https://nvd.nist.gov/vuln/detail/CVE-2023-29347) isn't a steam client vulnerability

Reply:
Correct. I've removed them. Thank you. 

Not sure where I got that info from, I think NVD.